playbooks/engineer-workflow.md

```diff
@@ -49,26 +49,41 @@ PR titles, because by default, we squash our commits when merging a PR, and this
 automatically ends up using the conventional commit format. Using the prefix in your branch name is not essential,
 but it's, well, nice.
 
-### Commits
+### Commit Messages
 
 > A well-crafted git commit message is the best way to communicate context about a change to fellow developers (and
 > indeed to [our] future selves). -[How to write a git commit message](https://chris.beams.io/posts/git-commit/)
 
-The code change itself should be internally consistent and coherent. If you're modifying a method signature,
-propagate the change to calling code. If you've implemented a new behavior, update test assertions to match. Except
-in rare cases where you intend to share a work-in-progress, commits should not leave tests in a failing state.
+In support of this, Artsy engineering adopted [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) as its commit message convention (see RFC [Best Practices for Naming and Merging PRs](https://github.com/artsy/README/issues/327)). We ecourage you to read the [spec](https://www.conventionalcommits.org/en/v1.0.0/#specification) for more detail but, in general, all commits should match the following format ([see examples](https://www.conventionalcommits.org/en/v1.0.0/#examples)): 
 
-Commit subjects should concisely describe the enhancement or fix in the imperative form ("Add tracking of email
-deliveries" or "Undo home page A/B test"). Git subcommands (like `git blame`, `revert`, `log`...) expect this
-format and become more transparent to use when you adhere to that convention.
+```
+<type>[optional scope]: <description>
 
-Expand on the subject to explain _why_ and add any helpful background or links in the commit body (separated by 1
-line). If your project uses GitHub issues, you can include keywords (like `closes #23`) to
-[close issues via commit](https://help.github.com/articles/closing-issues-via-commit-messages/).
+[optional body]
 
-Sometimes bad commit messages aren't apparent until later. Feel free to
-[squash](https://github.com/blog/2141-squash-your-commits) and rewrite commits on your working branch (even when
-it's in pull request form).
+[optional footer(s)]
+```
+
+Many Artsy engineers will include the Jira ticket id as the `scope`, which will automatically link the PR to the Jira ticket. A basic Artsy commit may look like: 
+
+`feat(FX-4093): curated marketing collections`
+
+The two main `types` of commits are `fix` and `feat`, however, commits with the following `types` are allowed: 
+- `build`
+- `chore`
+- `ci`
+- `docs`
+- `feat`
+- `fix`
+- `perf`
+- `refactor`
+- `revert`
+- `style`
+- `test`
```

Comment:
about the tooling that we currently have, how adjustable is it? if we wanted to start using a new type, say `potato`, for every PR that includes the a gif of a potato in a repo.
how easy/hard is that adjustment?

Reply:
This should just affect Peril and commitlint. For Peril, we just need to add it to the `danger` plugin. We did this [recently](https://github.com/artsy/peril-settings/pull/198) for `refactor`. For `commitlint`, looks like we would just need to add a rule in the config (I believe it's currently only in [Forque](https://github.com/search?q=org%3Aartsy+commitlint&type=code)). Not sure if there is more tooling I'm unaware of?